src/common.h

```diff
@@ -13,6 +13,7 @@
 #include "cluster.h"
 #include <stdio.h>
 #include <stdbool.h>
+#include "utils/arr_rm_alloc.h"
```

Comment:
Why is it needed here?

Reply:
It's a quite common header, so instead of adding it to all source files that need it, I just inserted it into `common.h` (which is included into all .c files by a build flag).